internal/ethapi/api.go

```diff
@@ -1028,15 +1031,81 @@ func DoCall(ctx context.Context, b Backend, args TransactionArgs, blockNrOrHash
 	}
 
 	// If the timer caused an abort, return an appropriate error message
+	timeoutMu.Lock()
 	if evm.Cancelled() {
+		timeoutErrors++
+
+		if timeoutErrors >= pprofThreshold {
+			timeoutNoErrors = 0
```

Comment:
Won't it be `timeoutErrors` here?

Reply:
I was thinking on resetting timeoutNoErrors here, because timeoutErrors happened.

Reply:
Got it, makes sense.

Reply:
Got it, makes sense.